src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml

```diff
@@ -0,0 +1,205 @@
+﻿<UserControl x:Class="Dynamo.UI.Controls.IncanvasSearchControl"
```

Comment:
I have had a word with @elayabharath and it seems like `InCanvasSearchControl` is easier to read (and much easier for my eyes) than `IncanvasSearchControl`, so can you rename it? Thanks :)